game/lib/global_leaderboard/podium_widgets.dart

```diff
@@ -15,89 +23,63 @@ var pointsStyle = TextStyle(
-      ? 'assets/icons/podium2highlighted.svg'
-      : 'assets/icons/podium2red.svg';
-
-  return Center(
-      child: Container(
-          width: 106,
-          height: 86,
-          child:
-              Stack(alignment: FractionalOffset(0.5, 0.63), children: <Widget>[
-            SvgPicture.asset(
-              svgAssetPath,
-              semanticsLabel: '2nd Podium',
-            ),
-
-            SizedBox(
-              width: 60,
-              height: 29,
-              child: FittedBox(
-                alignment: Alignment.center,
-                fit: BoxFit.scaleDown,
-                child: Text(points.toString() + " PTS", style: pointsStyle),
-              ),
-            ),
-            // Text(points.toString() + " PTS"
-          ])));
-}
+Widget PodiumBlock(context, int position, List scoreList) {
+  Map<int, String> svgAssetPaths = {
+    0: 'assets/icons/blank_podiums.svg',
+    1: 'assets/icons/gold_podiums.svg',
+    2: 'assets/icons/silver_podiums.svg',
+    3: 'assets/icons/bronze_podiums.svg',
+  };
 
-Widget ThirdPodium(context, int points, bool isUser) {
-  String svgAssetPath = isUser
-      ? 'assets/icons/podium3highlighted.svg'
-      : 'assets/icons/podium3red.svg';
-  return Center(
-      child: Container(
-          width: 106,
-          height: 62,
-          child:
-              Stack(alignment: FractionalOffset(0.5, 0.75), children: <Widget>[
-            SvgPicture.asset(
-              svgAssetPath,
-              semanticsLabel: '3rd Podium',
-            ),
+  String svgAssetPath = svgAssetPaths[position] ?? svgAssetPaths[0]!;
 
-            SizedBox(
-              width: 60,
-              height: 29,
-              child: FittedBox(
-                alignment: Alignment.center,
-                fit: BoxFit.scaleDown,
-                child: Text(points.toString() + " PTS", style: pointsStyle),
-              ),
-            ),
-            // Text(points.toString() + " PTS"
-          ])));
+  return Padding(
+    padding: const EdgeInsets.only(bottom: 10.0),
+    child: Stack(alignment: Alignment.bottomCenter, children: <Widget>[
+      SvgPicture.asset(
+        svgAssetPath,
+        width: 360,
+        height: 178,
+      ),
+      Positioned(
+        left: 27,
+        bottom: 12,
+        child: SizedBox(
+          width: 60,
+          height: 29,
+          child: FittedBox(
+            alignment: Alignment.center,
+            fit: BoxFit.scaleDown,
+            child: Text(scoreList[1].toString() + " PTS", style: pointsStyle),
+          ),
+        ),
+      ),
+      Positioned(
+        left: 150,
+        bottom: 25,
+        child: SizedBox(
+          width: 60,
+          height: 29,
+          child: FittedBox(
+            alignment: Alignment.center,
+            fit: BoxFit.scaleDown,
+            child: Text(scoreList[0].toString() + " PTS", style: pointsStyle),
+          ),
+        ),
+      ),
+      Positioned(
+        right: 27,
+        bottom: 6,
```

Comment:
I'm a little concerned about these hard coded positioning values. They looked fine to me on an Android emulator but I'm quite sure it would look off for different phones.